client/src/components/testUtilities.tsx

```diff
@@ -140,7 +137,7 @@ export const withMockFetch = async (
       // eslint-disable-next-line no-console
       console.error('Unexpected extra fetch request:\n', { url, options })
     }
-    return new Response(JSON.stringify({}))
+    return new Response(JSON.stringify(null))
```

Comment:
the changes in this file are just cleanup that i ended up doing while trying to figure out why some tests were passing after i modified them to fail. they ended up not being necessary since I decided to not have the frontend send the CSRF token header if no CSRF token cookie was set, but I figured I'd include them anyway